### PR TITLE
[Docs] Update cloud logos

### PR DESCRIPTION
**Pull request by Michaelvll**
Add runpod to the cloud logos

Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,8 +67,8 @@ pip install -U "skypilot[aws,gcp,azure,oci,lambda,runpod,ibm,scp,kubernetes]"  #
 Current supported providers (AWS, Azure, GCP, OCI, Lambda Cloud, RunPod, IBM, Samsung, Cloudflare, any Kubernetes cluster):
 <p align="center">
   <picture>
-    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/skypilot-org/skypilot/master/docs/source/images/cloud-logos-dark.png">
-    <img alt="SkyPilot" src="https://raw.githubusercontent.com/skypilot-org/skypilot/master/docs/source/images/cloud-logos-light.png" width=80%>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/skypilot-org/skypilot/update-cloud-logos/docs/source/images/cloud-logos-dark.png">
+    <img alt="SkyPilot" src="https://raw.githubusercontent.com/skypilot-org/skypilot/update-cloud-logos/docs/source/images/cloud-logos-light.png" width=80%>
```
Comment (romilbhardwaj) [inline README.md:71]: Should we change this back to `master`?

Comment (romilbhardwaj) [inline README.md:71]: Minor suggestion: should we move the IBM cloud a few pixels up and to the right? To create an even margin with Azure and Runpod logos.

**Review by romilbhardwaj — APPROVED** [review]
Thanks @Michaelvll!

File: README.md (hunk 1 above)
Comment (Michaelvll) [reply README.md]: Great call! Updated. Thanks!

**Review by romilbhardwaj — APPROVED** [review]
Thanks @Michaelvll!

**Pull request merged**